db/schema.rb

```diff
@@ -72,6 +72,7 @@
     t.integer  "align",                        default: 0
     t.datetime "created_at",                                 null: false
     t.datetime "updated_at",                                 null: false
+    t.string   "color"
```

Comment:
The schema dump does not represent all the migration changes, the `default` and `null` are absent. Please dump it again.

Reply:
Fixing